src/scss/components/seekbar/_seek-bar.scss

```diff
@@ -105,11 +108,30 @@ $seekbar-height: .3125em;
         transition-timing-function: linear;
         width: $marker-width;
 
+        transition: height 0.3s ease, border-radius 0.3s ease, transform 0.1s linear;
+
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+
+        &:hover {
+          height: $marker-indicator-height;
+          border-radius: $marker-indicator-border-radius;
+        }
+
         > .#{$prefix}-seekbar-marker-image {
           height: $seekbar-height * 4;
           position: absolute;
           transform: translate(-50%, calc(-100% - .2em));
         }
+
+        > .#{$prefix}-seekbar-marker-indicator {
+          height: $marker-indicator-height;
+          width: $marker-width;
+          background-color: $color-primary;
```

Comment:
Are we sure about this color?
If you are playing before a range, you don't really see the buffer indicator anymore

![image](https://github.com/user-attachments/assets/dc4e5b9d-6e5f-4382-b040-c144ffc67eda)

Reply:
Just X-checked, and it seems that's already the way we currently handle it as well:

![image](https://github.com/user-attachments/assets/ba74eaa3-2676-41c1-9a0d-ec71a782a7d9)

Reply:
Just X-checked, that seems to be the case already

![image](https://github.com/user-attachments/assets/5370735b-b9b8-4400-be2c-92f77d11a4a7)